Susi/MainViewController.swift

```diff
@@ -371,6 +380,94 @@ class MainViewController: UICollectionViewController, UICollectionViewDelegateFl
         }
     }
     
+    // Show image picker to set/reset wallpaper
+    func showImagePicker() {
+        if UIImagePickerController.isSourceTypeAvailable(UIImagePickerControllerSourceType.savedPhotosAlbum) {
+            imagePicker.allowsEditing = false
+            imagePicker.delegate = self
+            imagePicker.sourceType = UIImagePickerControllerSourceType.savedPhotosAlbum
+            self.present(imagePicker, animated: true, completion: nil)
+        } else {
+            // Show error dialog if no photo is available in photo album
+            let errorDialog = UIAlertController(title: "Image Pick Error", message: "Cannot access photos in your Albums", preferredStyle: UIAlertControllerStyle.alert)
```

Comment:
These a re not generic constants(will be only used by this file),should i move to constants or make then constant in this file only?